src/todo/todo.lua

```diff
@@ -335,6 +335,16 @@ function todo.on_gui_click(event)
     elseif (element.name == "todo_cancel_button") then
         element.parent.parent.destroy()
         todo.refresh_task_table(player)
+    elseif (string.find(element.name, "todo_delete_button")) then
+        local id = todo.get_task_id_from_element_name(element.name, "todo_delete_button_")
+
+        todo.create_delete_confirmation_button(element, id)
+    elseif (string.find(element.name, "todo_confirm_deletion_button")) then
+        local id = todo.get_task_id_from_element_name(element.name, "todo_confirm_deletion_button_")
+
+        todo.delete_task(id)
+        element.parent.parent.destroy()
```

Comment:
There should be a comment here on what is element.parent.parent

Reply:
true.